frontend/spa-assemble-config.json

```
@@ -41,6 +41,7 @@
     "@openmrs/esm-ward-app": "next",
     "@openmrs/esm-stock-management-app": "next",
     "@openmrs/esm-billing-app": "next"
+    "@openmrs/esm-reports-app": "next"
```

```suggestion
    "@openmrs/esm-billing-app": "next",
    "@openmrs/esm-reports-app": "next"
```

Please add the missing comma.